libscript/src/module-math.cpp

```diff
@@ -369,6 +369,10 @@ extern "C" MC_DLLEXPORT void MCMathEvalConvertToBase10(MCStringRef p_operand, in
         else
             r_output = t_negative ? -t_result : t_result;
     }
+	else if (t_error)
```

Comment:
This isn't quite right... If MCMathConvertToBase10 returns false then it means it threw an error (failure to allocate memory)... If it returns 'true' then 't_error' will indicate whether it managed to convert or not.